spec/lib/vmdb/configuration_encoder_spec.rb

```diff
@@ -44,44 +44,50 @@
     end
   end
 
-  VMDB_CONFIG_STRINGS = <<-YAML
+  let(:vmdb_config_strings) do <-YAML
   one:
     two: two
     three:
       four: four
   YAML
+  end
+
 
-  VMDB_CONFIG_SYMBOLIZED = <<-YAML
+  let(:vmdb_config_symbolized) do <-YAML
   :one:
     :two: two
     :three:
       four: four
   YAML
+  end
 
-  VMDB_CONFIG_MIXED = <<-YAML
+  let(:vmdb_config_mixed) do <-YAML
   :one:
     two: two
     :three:
       four: four
   YAML
+  end
 
-  VMDB_CONFIG_NUMERICS = <<-YAML
+  let(:vmdb_config_numerics) do <-YAML
   :1:
     2: two
   3:
     "4":
       5: five
   YAML
+  end
 
-  VMDB_CONFIG_DIFFERENT_STRINGS = <<-YAML
+  let(:vmdb_config_different_strings) do <-YAML
   one:
     two: one
     three:
       four: one
   YAML
+  end
 
-  VMDB_CONFIG_NUMERICS_SYMBOLIZED_HASH = {:"1" => {:"2" => "two"}, :"3" => {:"4" => {5 => "five"}}}
-  VMDB_CONFIG_SYMBOLIZED_HASH = YAML.load(VMDB_CONFIG_SYMBOLIZED)
+  vmdb_config_numerics_symbolized_hash = {:"1" => {:"2" => "two"}, :"3" => {:"4" => {5 => "five"}}}
+  vmdb_config_symbolized_hash = YAML.load(vmdb_config_symbolized)
```

Comment:
IMHO Yes; more importantly they should be nested within the direct context they are used in ("symbolizes", below). (Not your original intention with this change, I know; just a quick win)